docs/loading-iris-level-2-raster-and-sji-data.rst

```diff
@@ -0,0 +1,37 @@
+.. _loading_iris_level_2_raster_and_sji_files:
+
+=======================================================
+Loading IRIS Level 2 Raster and SJI Data Files Together
+=======================================================
+
+Using the custom IRIS importer
+------------------------------
+Using the custom IRIS ``glue`` data importer, one can handle the loading of different raster
+and SJI files of IRIS Level 2 data for the same observation with ease. To begin, point the
+cursor to the "File" dropdown menu and the choose "Import Data", and after which choose the
+"Import IRIS OBS Directory" option. Like in the image below:
+
+.. image:: images/loading-iris-data-1.png
+   :width: 800
+   :alt: The "Import IRIS OBS Directory" option in ``glue``
+
+Take care to make sure all the IRIS data files for the same
+observations you would like to investigate with ``glue`` are all located in the same directory.
+Then, a popup dialog box would allow you to choose the desired observation directory containing
+the "raster" and "SJI" files of interest. Upon clicking "Open" on the lower right of the dialog
+box, a VizieR importer would pop up with the heading "Load IRIS Observation", with the available
+data sets listed in the respective "raster" and "SJI" fields, as shown in the image that follows:
+
+.. image:: images/loading-iris-data-2.png
+   :width: 800
+   :alt: Loading Raster and SJI data together with the VizieR importer
+
+After successfully loading the chosen data files, if autolinking if available, you will be prompted
+to link up the files concerned. A dialog box like what follows would pop up:
+
+.. image:: images/loading-iris-data-3.png
+   :width: 800
+   :alt: Autolinking option for the IRIS OBS importer
+
+Finally, you are all set to examine the IRIS Level 2 data using ``glue`` with the ``glue-solar``
+plugin. Voila!
```

Comment:
Could we get an example of the final result?

Reply:
Sure, that could be another RST file or the same one... I only included the details for the loading since that is what is listed in our plans for `glue-solar`. But I could open another PR to discuss how to use `glue-solar` to investigate IRIS Level 2 "raster" and "SJI" data like what I have done recently.

Reply:
Would the load not mean an image viewer is created at the end of the process?

Reply:
Not really, the loader and viewer are two distinct parts from what I can tell...

Reply:
So we can load the data and then do nothing with it?

Reply:
Yup, think so... Unless you know what to do next, otherwise it is easy to lose directions in what needed to be done to meaningfully look at the "raster" and "SJI" data from the same observation together.

Reply:
Sure but I am not sure that an example that just loads data is all that useful.

Reply:
I totally agree, which is the reason why I had been playing around with some IRIS level 2 data to get a sense about what I have been doing... I will writeup the details regarding the use of glue-solar for IRIS data after the data have been properly loaded soon.

Reply:
Thanks!

Reply:
Hence the need to look up some spectral lines and determine what their uses are so I could include these in our piece, as I have previously mentioned on Riot IRC earlier.

Reply:
My pleasure @nabobalis! 😃